middleware/jwt_test.go

```diff
@@ -394,6 +396,77 @@ func TestJwt(t *testing.T) {
 	tests.Assert(t, r.StatusCode == http.StatusOK)
 	tests.Assert(t, called == true)
 }
+func TestJwtIgnoreIAT(t *testing.T) {
```

Comment:
I guess gofmt doesn't care about this, but I do: there should be an empty line between the closing curly brace of the previous function and the start of the new one.